exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/exporter.py

```diff
@@ -118,22 +120,55 @@ def get_resource_data(
     return _get_resource_data(sdk_resource_scope_data, resource_class, name)
 
 
-def _load_credential_from_file(filepath) -> ChannelCredentials:
+def _read_file(file_path: Optional[str]) -> Optional[bytes]:
```

Comment:
Does this type need to be updated? Doesn't look like `file_path` is actually optional.

Reply:
Good point.. Forgot to change this

Reply:
Fixed!